frontend/src/components/common/LoginBtn/RegisterModal/RegisterModal.jsx

```diff
@@ -0,0 +1,168 @@
+            <form onSubmit={handleSubmit}>
+              <div className={styles.dataContainer}>
+                <div className={styles.nameContainer}>
+                  <label className={styles.labelData} htmlFor="name">
+                    Ім’я {submitted && !name && <span>*</span>}
+                  </label>
+
+                  <input
+                    id="name"
+                    className={styles.dataInput}
+                    type="text"
+                    placeholder="Введіть ваше ім’я"
+                    onChange={(e) => setName(e.target.value.trim())}
+                  />
+                  {submitted && !name && (
+                    <p className={styles.errorMessage}>Введіть ваше ім`я</p>
+                  )}
+                </div>
+                <div className={styles.emailContainer}>
+                  <label className={styles.labelData} htmlFor="email">
+                    Ел. пошта {submitted && !email && <span>*</span>}
+                  </label>
+
+                  <input
+                    id="email"
+                    className={styles.dataInput}
+                    type="email"
+                    placeholder="Введіть електронну пошту"
+                    value={email}
+                    onChange={(e) => {
+                      setEmail(e.target.value.trim());
+                      setEmailExistsError('');
+                    }}
+                  />
+                  {submitted && emailExistsError && (
+                    <p className={styles.errorMessage}>{emailExistsError}</p>
+                  )}
+                  {submitted && !email && (
+                    <p className={styles.errorMessage}>
+                      Введіть електронну пошту
+                    </p>
+                  )}
+                  {submitted && email && !validateEmail(email) && (
+                    <p className={styles.errorMessage}>
+                      Невірний формат електронної пошти
+                    </p>
+                  )}
+                </div>
+
+                <div className={styles.paswordContainer}>
+                  <label className={styles.labelData} htmlFor="password">
+                    Пароль {submitted && !password && <span>*</span>}
+                  </label>
+
+                  <input
+                    id="password"
+                    name="password"
+                    className={styles.dataInput}
+                    type="password"
+                    placeholder="Введіть ваш пароль"
+                    value={password}
+                    onChange={(e) => setPassword(e.target.value)}
+                  />
+                  {submitted && !password && (
+                    <p className={styles.errorMessage}>Введіть пароль</p>
+                  )}
+                  {submitted && password && !validatePassword(password) && (
+                    <p className={styles.errorMessage}>
+                      Пароль має бути не менше 8 символів, містити латинські
```

Comment:
Did we agree to use popup window for this info? like title attribute

Reply:
The design is not ready yet